README.md

```diff
@@ -8,20 +8,19 @@ Contributing
 ------------
 
 #### Getting started
+
 1. Install [Nodejs](http://nodejs.org/) and [npm](https://www.npmjs.org/)
-2. Install [grunt-cli](http://gruntjs.com/installing-grunt) and [bower](http://bower.io/) by running `npm install -g grunt-cli bower` (may need to be run with sudo)
-3. Install [ruby](https://www.ruby-lang.org/en/)
-4. Install bundler `gem install bundler`
-5. Install [Go](https://golang.org/dl/) per [steps 1 - 3](https://github.com/openshift/origin/blob/master/CONTRIBUTING.adoc#develop-locally-on-your-host)
-6. Install dev dependencies by running `hack/install-deps.sh`
-7. Launch the console and start watching for asset changes by running `grunt serve`. This should open <https://localhost:9000/> in your default browser.
+1. Install [grunt-cli](http://gruntjs.com/installing-grunt) and [bower](http://bower.io/) by running `npm install -g grunt-cli bower` (may need to be run with sudo)
+1. Install [Go](https://golang.org/dl/) per [steps 1 - 3](https://github.com/openshift/origin/blob/master/CONTRIBUTING.adoc#develop-locally-on-your-host) in the OpenShift contributing guide.
```

Comment:
Removed reference to go, added a comment about `oc cluster up`, tho I'm sure anyone reading this would be uncertain about that command atm.